src/redux/actions/userActions.js

```diff
@@ -111,3 +111,39 @@ export function bootstrapUser (user) {
         user
     };
 }
+
+export function deleteRoute (routeId) {
```

Comment:
I have changed all the three PR's to use updateroute :-). Have a look when you have the time.